src/components/nationalDashboard.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import styles from '../styles/dashboard.module.less';
+import IframeResizer from 'iframe-resizer-react';
+import { Select } from 'antd';
+import { DataSourceType, dataSources } from '../content/NationalDashboard';
```

Comment:
since `NationalDashboard` is not a component, it is better to keep it lower case `nationalDashboard`